packages/sdk-js/src/handlers/server.ts

```diff
@@ -1,5 +1,30 @@
 import { AxiosInstance } from 'axios';
 
+export type ServerInfo = {
```

Comment:
There's more info in here when you're authenticated

Reply:
added